tests/smtp-file-data-01/test.yaml

```diff
@@ -0,0 +1,74 @@
+requires:
+  min-version: 6
```

Comment:
Q: Could you please tell why this was added? Did you find a backport missing only in 5?

Reply:
Yes. It without this the CI fails.

Reply:
I suggested that she added it, as even after rebasing the tests were still failing for 5. I'm sorry if that was a misdirection...